docs/pages/web/toast.js

```diff
@@ -194,115 +430,171 @@ If  confirmation toast's text with more complex style is required, such as bold
-      accessibilityLabel: '',
-      text: 'Home decor',
-      href: 'http://www.pinterest.com',
-      onClick: () => {},
-    }}
-    button={<Button key="button-key" text="Undo" size="lg" />}
-    text="Saved to"
-    thumbnail={{ image:
-      <Image
-        alt="Modern ceramic vase pin."
-        naturalHeight={564}
-        naturalWidth={564}
-        src="https://i.ibb.co/Lx54BCT/stock1.jpg"
-      />
+    button={<Button key="button-key" text="Undo" />}
+    text="Save the link from your clipboard?"
+    thumbnail={{
+      icon: ( <Icon
+        accessibilityLabel="Go to next steps"
+        icon="link"
+      /> )
     }}
   />
+</Flex>
+`}
+            description="For when an icon is needed to represent content that isn’t a pin or a profile."
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection title="Dismissable">
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+<Flex justifyContent="center" width="100%">
   <Toast
-    button={<Button key="button-key" text="Undo" size="lg" />}
-    text="Switched to Silverio Ibrahim"
-    thumbnail={{ avatar:
-      <Avatar
-        src="https://i.ibb.co/ZfCZrY8/keerthi.jpg"
-        name="Keerthi"
-      />
-    }}
+    text="Your Pin was saved"
+    dismissButton={{ onDismiss: () => {} }}
   />
-</Flex>
+</Flex>`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection
+        name="Duration and timing"
+        description={`
+Toasts should be on screen for a minimum of 5s; this gives most people enough time to read and act. Please note that a separate Toast manager must be implemented in order to handle duration and animation.
+
+Once a toast is triggered, allow for a cooldown period of about 7 seconds before the toast can be triggered again. This will prevent multiple toasts from appearing.
 `}
+      />
+
+      <MainSection name="Writing">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description={`
+- Consider internationalization and how other languages may be take up more space
+- Be brief and concise
+- Use conversational language`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description={`
+- Use lengthy, technical jargon or local idioms that will be hard to translate to other languages`}
           />
         </MainSection.Subsection>
       </MainSection>
 
       <QualityChecklist component={generatedDocGen?.displayName} />
+
+      <MainSection name="Related">
+        <MainSection.Subsection
+          description={`
+      **[Alert Modal](/web/modalalert)**
+      An AlertModal is a simple modal dialog used to alert a user of an issue, or to request confirmation after a user-generated action. AlertModal overlays and blocks Page content until it is dismissed by the user.
+
+      **[Generic Modal](/web/modal)**
```

Comment:
I'd avoid the word "Generic" here since it's not part of the component's official name